Website/release.config

```diff
@@ -124,7 +124,7 @@
     <trust level="Full" originUrl=".*" />
     -->
     <!-- set debugmode to false for running application -->
-    <compilation debug="false" strict="false" targetFramework="4.5">
+    <compilation debug="false" strict="false" targetFramework="4.7.2">
```

Comment:
Do we already have code to update the `web.config` on upgrades?

Reply:
This was missed when I put in the 4.7.2 PR. Where does the upgrade code for the web.config exist? If it is not in there we need to make sure it is included.

Reply:
@ohine would this be `Upgrade.cs` code that calls an XML Merge template?